src/Symfony/Abstraction/README.md

```diff
@@ -0,0 +1,66 @@
+Symfony Abstractions
+====================
+
+A set of standards maintained by the Symfony community and extracted out of
+established and working solutions: the standalone Symfony components.
+
+Design Principles
+-----------------
+
+ - standards are split by domain, each in their own sub-namespace;
+ - each standard forms a closed abstraction, defined in terms of PHP interfaces,
+   traits, normative docblocks and reference test suite when applicable;
+ - all abstractions must have a proven implementation to enter this repository;
+ - they must be backward compatible with existing Symfony components.
+
+FAQ
+---
+
+### How to use this package?
+
+The abstractions in this packages are useful to achieve loose coupling and
+interoperability. E.g. by using the provided interfaces as type hints, you will
+allow reusing any implementation that matches their contracts. It could be a
+Symfony component, or another one provided by the community or just by you.
+
+Depending on their semantics, some interfaces can be combined with autowiring to
+seamlessly inject a service in your classes.
+
+Others might be useful as labeling interfaces, to hint about a specific behavior
+that could be enabled by using autoconfiguration or manual service tagging (or
+any other means provided by your framework.)
```

Comment:
Are these 2 paragraphs needed? Especially with only resettable, they don't seem to have a lot of meaning yet.

Reply:
the next PRs will make fully needed :)